1_fetch.yml

```diff
@@ -32,14 +32,16 @@ targets:
       - 1_fetch/out/site_stats.rds.ind
       - 1_fetch/out/dv_data.rds.ind
       - 1_fetch/out/dv_data_fixed_gh.rds.ind
-      - 1_fetch/out/dv_historic.ind
+      #- 1_fetch/out/dv_historic.rds.ind
```

Comment:
i also had to delete this file locally, but I didn't check in that change. should I @lindsayplatt ?

Reply:
nope, I think this is good